akka-http-circe/src/main/scala/de/heikoseeberger/akkahttpcirce/CirceSupport.scala

```diff
@@ -22,9 +22,11 @@ import akka.http.scaladsl.model.MediaType
 import akka.http.scaladsl.model.MediaTypes.`application/json`
 import akka.http.scaladsl.unmarshalling.{ FromEntityUnmarshaller, Unmarshaller }
 import akka.util.ByteString
-import cats.data.NonEmptyList
+import cats.data.{ NonEmptyList, ValidatedNel }
 import cats.syntax.show.toShow
+import cats.syntax.either._
```

Comment:
Please no wildcard imports at top level. Change to specific imports or move import statement to narrowest scope possible.

Reply:
Updated.